lib/seek/permissions/publishing_permissions.rb

```diff
@@ -42,10 +51,15 @@ def publish!(comment = nil, force = false)
             policy.access_type = Policy::ACCESSIBLE
             policy.sharing_scope = Policy::EVERYONE # anything but ALL_USERS
             policy.save
-            # FIXME: may need to add comment
+
             resource_publish_logs.create(publish_state: ResourcePublishLog::PUBLISHED,
                                          user: User.current_user,
                                          comment: comment)
+
+            unless contributor.is_asset_gatekeeper_of?(self)
+              resource_publish_logs.find_or_initialize_by(publish_state: ResourcePublishLog::WAITING_FOR_APPROVAL, user: contributor.user )
```

Comment:
I'm confused by this, because a log has already been created above. Couldn't this create a 2nd duplicate log ?